token/cli/src/command.rs

```diff
@@ -3472,6 +3472,7 @@ pub async fn process_command<'a>(
             let member_address = value_t!(arg_matches, "member_address", Pubkey).ok();
 
             let transfer_fee = arg_matches.values_of("transfer_fee").map(|mut v| {
+                println_display(config,"transfer_fee flag has been deprecated".to_string());
```

Comment:
nit: We can give a bit more info
```suggestion
                println_display(config,"transfer-fee has been deprecated and will be removed in a future release. Please specify --transfer-fee-basis-points and --transfer-fee-maximum-fee with a UI amount".to_string());
```